schemas/com.snowplowanalytics.mobile/remote_config/jsonschema/1-0-1

```diff
@@ -0,0 +1,225 @@
+{
+  "$schema" : "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self" : {
+    "vendor" : "com.snowplowanalytics.mobile",
+    "name" : "remote_config",
+    "version" : "1-0-1",
+    "format" : "jsonschema"
+  },
+  "description": "The configuration file for the Snowplow mobile trackers.",
+  "type": "object",
+  "properties": {
+    "configurationVersion": {
+      "description": "Version number that identifies the current configuration. It has to be increased on each update.",
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 2147483647
+    },
+    "configurationBundle": {
+      "description": "The list of configurations for each tracker to configure.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "description": "Configuration objects",
+        "properties": {
+          "namespace": {
+            "description": "The namespace string that identifies the single tracker.",
+            "type": "string",
+            "maxLength": 4096
+          },
+          "networkConfiguration": {
+            "description": "Represents the network communication configuration allowing the tracker to be able to send events to the Snowplow collector.",
+            "type": ["object", "null"],
+            "properties": {
+              "endpoint": {
+                "description": "URL of the collector that is going to receive the events tracked by the tracker. The URL can include the schema/protocol (e.g.: http://collector-url.com). In case the URL doesn’t include the schema/protocol, the HTTPS protocol is automatically selected.",
+                "type": "string",
+                "maxLength": 2048
+              },
+              "method": {
+                "description": "The method used to send the requests (GET or POST).",
+                "type": ["string", "null"],
```

Comment:
Hey @matus-tomlein just a warning, the "null" on this line does not do what you think it does!  If you want this field to be nullable then you must also add `null` to the enum:

```
"enum": ["get", "post", null]
```

Otherwise, a null will get rejected during validation.

I know this is slightly surprising.  You can test it out though [using an online validator](https://www.jsonschemavalidator.net/).

Reply:
Thanks @istreeter, didn't know about this! Fixed.